Makefile

```
@@ -26,6 +26,10 @@ GOBIN ?= $$(go env GOPATH)/bin
 
 build: $(TOPDIR)/bin/amd64-linux/habitat $(TOPDIR)/bin/amd64-darwin/habitat
 
+build-ctrl: 
```

Since the outcome of this build rule is a physical artifact, we can name the rule the path to the compiled bin. This allows the build system to take advantage of pre-built targets if other targets depend on them.

Then, to keep it easy to run it, we can make the `build-ctrl` just a phony shortcut, that always forces a rebuild.

```suggestion

.PHONY: build-ctrl

build-ctrl: clean-node-ctrl $(TOPDIR)/bin/node-ctrl

clean-build-ctrl:
    rm $(TOPDIR)/bin/node-ctrl

$(TOPDIR)/bin/node-ctrl:
     go build -o $(TOPDIR)/bin/node-ctrl $(TOPDIR)/cmd/node_ctrl/main.go
     
 clean:: clean-build-ctrl
```
The last line makes it so `make clean` includes cleaning up the CLI as well.

---

Did this!